src/map/entities/battleentity.cpp

```diff
@@ -151,7 +151,7 @@ bool CBattleEntity::isSitting()
 
 /************************************************************************
  *                                                                       *
- *  Пересчитываем максимальные значения hp и mp с учетом модификаторов   *
+ *  Recalculate hp and mp maximum values, taking into account modifiers  *
```

Comment:
as an example: this one is actually mildly useful, because it lets the reader know this function INCLUDES modifiers before they go into the actual code to fid that out